src/test/resources/maven/backport-util-concurrent/backport-util-concurrent-java12/3.1/backport-util-concurrent-java12-3.1.pom

```diff
@@ -5,7 +5,6 @@
     <version>3.1</version>
     <packaging>jar</packaging>
     <name>Backport of JSR 166 for Java 1.2 and 1.3</name>
-    <url>http://backport-jsr166.sourceforge.net/</url>
```

Comment:
Hope it's ok that I edited this and the following two to have test cases without adding more to the repo.

Reply:
Why remove this? Just update the tests :P